gcloud-java-examples/README.md

```diff
@@ -82,7 +82,7 @@ To run examples from your command line:
 
   * Here's an example run of `StorageExample`.
 
-    Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud Storage API is enabled and that you have a bucket.  Also ensure that you have a test file (`test.txt` is chosen here) to upload to Cloud Storage stored locally on your machine.
+    Before running the example, go to the [Google Developers Console][developers-console] to ensure that "Google Cloud Storage" and "Google Cloud Storage API" are enabled and that you have a bucket.  Also ensure that you have a test file (`test.txt` is chosen here) to upload to Cloud Storage stored locally on your machine.
```

Comment:
Do you think it's worth to use "Google Cloud Storage JSON API" to be consistent with how it's called in the console?